packages/apps/web-viewer-test/src/components/home/AuthConfigHome.tsx

```diff
@@ -89,6 +94,36 @@ export const AuthConfigHome: React.FC = () => {
     }
   };
 
+  const viewConfiguration = (viewPort: ScreenViewport) => {
```

Comment:
isn't this the default behavior if you don't provide `standardViewId`?

Reply:
which line? The whole function? It would not be the default if the model has a default view id.....This is basically saying to fit the view and set the standard rotation to Iso regardless

Reply:
Ah yes thats right, sorry disregard this comment. I guess if anything you could also change this to be .finally as well